scipy/interpolate/_fitpack_repro.py

```diff
@@ -501,12 +575,85 @@ def __call__(self, p):
         _dierckx.qr_reduce(AB, offset, nc, QY, startrow=nc)
 
         # solve for the coefficients
-        c = _dierckx.fpback(AB, nc, QY)
+        c, _, fp = _dierckx.fpback(AB, nc, self.x, self.y, self.t, self.k, self.w, QY)
 
         spl = BSpline(self.t, c, self.k)
-        residuals = _compute_residuals(self.w**2, spl(self.x), self.y)
+        self.spl = spl   # store it
+
+        return fp - self.s
+
+# Solver class for f(p) for periodic splines
```

Comment:
Please explain the difference to `F`. What's sorely needed is an explanation of A1, A2, Z and H1, H2.
Ideally, have something similar to the docstring of F.